### DISCUSSION TITLE
Should stopping current task overwrite "from"?

### DISCUSSION THREAD

**Discussion by virtulis** [AlexandrePTJ/kemai#127] [General]
I'm not sure this is a bug and looking at the source not sure if this is an easy change.

Basically, sometimes I forget to start the task for some time. So when I do that I used to go the web UI and change the from time manually. This works fine if I do that after stopping, but turns out if I edit while running it *seems to work* but then Kemai sends a patch request with both To and From times (or so it seems from the "audit log"). Apparently I lost a couple of hours this month like that, oops :blush: 

I'm wondering how much of a pain it is to avoid re-sending the From time when stopping, since only way it can change is from elsewhere anyway?

Alternatively, would things break if the "Started at" input was not disabled while the task is going?

I can probably try doing either of those things but wondering if there's anything I might break with that.

Sorry about the fuss about such a minor issue, the app has been very reliable otherwise, thank you for your work :)

**Comment by AlexandrePTJ** [comment]
It is somekind of a bug. Kemai should update itself with changes from web UI.
I think that both `From` and `End` time are mandatory for the API. I will fix it anyway.

**Comment by AlexandrePTJ** [comment]
Issue created #129
